src/main/java/icbm/classic/content/reg/ItemReg.java

```diff
@@ -82,8 +83,15 @@ public static void registerItems(RegistryEvent.Register<Item> event)
         event.getRegistry().register(new ItemGrenade().setName("grenade").setCreativeTab(ICBMClassic.CREATIVE_TAB));
         event.getRegistry().register(new ItemBombCart().setName("bombcart").setCreativeTab(ICBMClassic.CREATIVE_TAB));
         event.getRegistry().register(new ItemBase().setName("poisonPowder").setCreativeTab(ICBMClassic.CREATIVE_TAB)); //TODO fix name _
-        event.getRegistry().register(new ItemBase().setName("sulfurDust").setCreativeTab(ICBMClassic.CREATIVE_TAB)); //TODO fix name _
-        event.getRegistry().register(new ItemBase().setName("saltpeter").setCreativeTab(ICBMClassic.CREATIVE_TAB));
+        Item sulfurItem = new ItemBase().setName("sulfurDust").setCreativeTab(ICBMClassic.CREATIVE_TAB);
+        event.getRegistry().register(sulfurItem); //TODO fix name _
+        OreDictionary.registerOre("dustSulfur", sulfurItem);
```

Comment:
Is is safe to add it to the registry at this stage? Especially given replacements for items are a thing?